win32ss/user/user32/controls/edit.c

```diff
@@ -351,9 +342,10 @@ static INT EDIT_CallWordBreakProc(EDITSTATE *es, INT start, INT index, INT count
 	    {
 		EDITWORDBREAKPROCW wbpW = (EDITWORDBREAKPROCW)es->word_break_proc;
 
-		TRACE_(relay)("(UNICODE wordbrk=%p,str=%s,idx=%d,cnt=%d,act=%d)\n",
+		TRACE_(relay)("\1Call wordbreakW %p (str=%s,idx=%d,cnt=%d,act=%d)\n",
```

Comment:
`\1` ? What is Wine trying to do here?

Reply:
https://source.winehq.org/git/wine.git/01676c5dd6f9fb61b9c4e52595f6e75e21050009